Automate backups should be initiated outside root-controlled directories

sean-horn opened the pull request:
root-only working directories cause failures during `automate-ctl create-backup` runs as the chpst command immediately tries to change into the user chef-pgsql's home directory and fails, causing follow on problems with the backup.

Zendesk 15110

jamescott commented:
@fishnix Great catch. I have submitted this to our Delivery-hosted repo and the change will be on the site sometime later today. Thanks!

jamescott commented:
... and it's updated (~25 days ago). Closing the PR. Thanks again!

sean-horn committed:
Automate backups should be outside root

root-only working directories cause failures during `automate-ctl create-backup` runs as the chpst command immediately tries to change into the user chef-pgsql's home directory and fails, causing follow on problems with the backup.

Commit:
Minor copyedit and replicated shared content changes

Signed-off-by: David Wrede <dwrede@chef.io>

davidwrede commented:
![tenor-129673764](https://user-images.githubusercontent.com/1634752/27982650-24f2bc4c-635c-11e7-9cb0-4d8b7f89ec20.gif)
